app/js/app/modules/domready.js

```diff
@@ -0,0 +1,19 @@
+const EVENT_DOM_READY = 'DOMContentLoaded'
```

Comment:
This is a convention for event name constants, probably not consistent with everything else though.